addons/portal/static/src/scss/bootstrap_overridden.scss

```diff
@@ -15,3 +15,8 @@ $font-size-sm: (12 / 16) * 1rem;
 
 $btn-padding-y-sm: (1 / 16) * 1rem;
 $btn-padding-x-sm: (5 / 16) * 1rem;
+
+// Modals
+
+$modal-lg: 980px;
+$modal-md: 650px;
```

Comment:
Maybe add those values in web/primary_variable with `$o-modal-lg` and use those in both bootstrap_overridden files.